tests/sentry/api/endpoints/test_user_permission_details.py

```diff
@@ -9,68 +13,149 @@ class UserDetailsTest(APITestCase):
 
     def setUp(self):
         super().setUp()
-        self.user = self.create_user(is_superuser=True)
```

Comment:
Changed the name of the default user in the tests from `self.user` to `self.superuser`